### ISSUE TITLE
Cannot Reproduce OneDiff results

### ISSUE THREAD

**Issue by chontipan** [AI-nstein/hoppity#14]
Hi, I run the provided code with the cooked dataset of OneDiff and the result does not reach the same as in the paper(i.e., total accuracy is 0.261 but I got around 0.25x). I run for 10000 epochs. I change comp_method in the provided code to ''bilinear" because the provided model may use bilinear. Could you please suggest me?

run_main.sh
![image](https://user-images.githubusercontent.com/9138402/101296345-de711900-385d-11eb-9048-6f5003ef4bf7.png)

find_best_model.sh
![image](https://user-images.githubusercontent.com/9138402/101296355-e8931780-385d-11eb-9a61-ebb9bf93018d.png)

eval.sh
![image](https://user-images.githubusercontent.com/9138402/101296366-fb0d5100-385d-11eb-9a1b-4078d5254f77.png)

**Comment by elizabethdinella** [comment]
Hi, yes we used `bilinear` to get the 26.13 result. Epoch 1030 was our best model. You are getting 25% overall acc on this configuration?

**Comment by chontipan** [comment]
Yes, I use these parameters to train the model. I changed only paths of directories and replaced 'bilnear'.

**Comment by elizabethdinella** [comment]
can you run eval on epoch 1030?

**Comment by chontipan** [comment]
![IMG_20201221_085421](https://user-images.githubusercontent.com/9138402/102729125-52391880-436a-11eb-9f11-2b6e82064848.jpg)
BTW,  I did not use rtx2080ti same as the paper.

**Comment by chontipan** [comment]
![image](https://user-images.githubusercontent.com/9138402/102736534-2759bf00-4380-11eb-80dd-3d63aa639222.png)
my GPU is gtx1060. I run eval on epoch 1030
